.ghci.repl

```diff
@@ -23,8 +23,16 @@
 :seti -Wno-name-shadowing
 :seti -Wno-safe
 :seti -Wno-unsafe
+
+-- ghc 8.6+
 :seti -Wno-star-is-type
```

Comment:
Could we ditch this particular clause entirely, since I think we've switched to using `Type` everywhere?

Reply:
We did this on `master` for 1.1, this PR is going to release 1.0.2.1.